eng/Publishing.props

```diff
@@ -2,5 +2,45 @@
 <Project>
    <PropertyGroup>
       <PublishingVersion>3</PublishingVersion>
+      <AutoGenerateSymbolPackages>false</AutoGenerateSymbolPackages> <!-- we don't need symbol packages for emsdk -->
    </PropertyGroup>
+
+  <PropertyGroup>
+    <_UploadPathRoot>wasi-sdk</_UploadPathRoot>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <PublishDependsOnTargets>$(PublishDependsOnTargets);_PublishInstallers</PublishDependsOnTargets>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <_InstallersToPublish Include="$(ArtifactsDir)**\*.wixpack.zip" />
```

Comment:
probably want to use `$(ArtifactsDir)\packages` or wherever this is produced so we don't accidentally grab files from obj